src/main/java/com/springweb/dv_spring_web_mongo/controller/RestController.java

```diff
@@ -0,0 +1,47 @@
+package com.springweb.dv_spring_web_mongo.controller;
+
+import com.springweb.dv_spring_web_mongo.model.Project;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.data.mongodb.core.query.Update;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@org.springframework.web.bind.annotation.RestController
+@RequestMapping("/api")
```

Comment:
All your api methods have `/projects` part in the url. You can specify it here:
@RequestMapping("/api/projects")

So you will need to remove it from correspond methods